test/e2e/common/node/sysctl.go

```diff
@@ -199,6 +155,27 @@ var _ = SIGDescribe("Sysctls [LinuxOnly] [NodeFeature:Sysctls]", func() {
 		gomega.Expect(err.Error()).NotTo(gomega.ContainSubstring("kernel.shmmax"))
 	})
 
+	ginkgo.It("should forbid unsafe sysctls [MinimumKubeletVersion:1.21]", func() {
```

Comment:
Can we keep the comments even if it's not conformance?

Reply:
OK, I add the comments back in #103906.